condensedinlinepanel/static/condensedinlinepanel/src/components/CardSet.tsx

```diff
@@ -178,7 +179,7 @@ export class CardSet extends React.Component<CardSetProps, {}> {
                 e.preventDefault();
                 return false;
             };
-            addButton = <button className="condensed-inline-panel__top-add-button button bicolor icon icon-plus" type="button" onClick={onClickAddButton}>Add</button>;
+            addButton = <button className="condensed-inline-panel__top-add-button button bicolor icon icon-plus" type="button" onClick={onClickAddButton}>Add {this.props.panelLabel}</button>;
```

Comment:
Works for me. I was mainly mimicking what inline panel did, but this also solves my problem.

We should then put Add as the default where the string constant is set.